src/pages/news.js

```diff
@@ -19,23 +19,25 @@ const NewsPage = props => {
   const { data } = props
   const { i18n, t } = useTranslation()
 
+  const news = withLanguage(i18n, data.allGoogleNews.edges[0].node, "gn")
```

Comment:
please don't use syntax like this. it will still have errors when either allGoogleNews/edges is empty
consider using something like 
```
const newsNode = _.get(data, 'allGoogleNews.edges[0].node', {})
const news = withLanguage(i18n, newsNode, "gn")
```